client/components/signup-form/index.jsx

```diff
@@ -591,6 +595,20 @@ class SignupForm extends Component {
 			<div className={ classNames( 'signup-form', this.props.className ) }>
 				{ this.getNotice() }
 
+				{ this.props.isSocialSignupEnabled &&
+					this.props.isSocialFirst && (
+						<Card className="signup-form__social logged-out-form">
+							<SocialSignupForm
+								handleResponse={ this.props.handleSocialResponse }
+								socialService={ this.props.socialService }
+								socialServiceResponse={ this.props.socialServiceResponse }
+							/>
+							<p>
+								{ preventWidows( this.props.translate( 'Or register with your e-mail address.' ) ) }
```

Comment:
We usually use `email` over `e-mail` and `sign up` over `register` in our copy.

Reply:
👍